components/bl-pdfViewer-component/styles/index.less

```diff
@@ -0,0 +1,98 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-pdfViewer-themeColor: @themePrimary;
+@bl-customComponent-pdfViewer-themeTextColor: @appTextColor;
+
+@bl-customComponent-pdfViewer-input-background-color: if(@isLightTheme, darken(#fff, 10%), lighten(#000, 15%));
+@bl-customComponent-pdfViewer-button-icon-size: 16px;
+
+@bl-customComponent-pdfViewer-no-data-icon-size: 40px;
+
+.bl-customComponent-pdfViewer {
+  color: @bl-customComponent-pdfViewer-themeTextColor;
+
+  .controls {
+    display: flex;
+    justify-content: space-between;
+
+    &.disabled {
+      opacity: 0.5;
+      cursor: auto;
+    }
+  }
+
+  .controls-button {
+    padding: 0;
+    border: 0;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 20px;
+    width: 60px;
+
+    fill: #fff;
+
+    border-radius: 4px;
+    background-color: @bl-customComponent-pdfViewer-themeColor;
+
+    cursor: pointer;
+
+    &:disabled {
+      opacity: 0.5;
+      cursor: auto;
+    }
+  }
+
+  .pdf-viewer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: auto;
+
+    margin-bottom: 10px;
+  }
+
+  .page-input {
```

Comment:
Write some styles to make input looks line page counter

For now it looks not so pretty
![image](https://user-images.githubusercontent.com/15347646/191717037-5193f50c-f970-45e7-add3-f6a8c25cd6e5.png)

Reply:
yes, I did as it was written in the comment below
"We could make an input transparent and for 4 digits with right content align - that will looks good, and could be written without JS"